services/lib.js

```diff
@@ -0,0 +1,29 @@
+const winston = require('winston')
+
+const logger = winston.createLogger({
```

Comment:
I'd prefer we always log to the console, no matter if it's in production or not 

later on we can add configs for files in the `cfg`, but I think for now it's best to log everything to the console and allow configurable levels/outputs later on